Added en text to community channel (#130)
DaoCloud/DaoCloud-docs: docs/en/community/ferry.md

# Ferry

Ferry is a Kubernetes multi-cluster communication component that eliminates communication differences between clusters

> as if they were in a single cluster, regardless of the network environment those clusters are in.

Reviewer: ```suggestion
as if they were in a single cluster, regardless of their network environments.
```
